### PR TITLE
ci: add extra json for cicd

### PR DESCRIPTION
**Pull request by dbrasseur-aneo**
When deploying using all in one, changing parameters programatically is difficult. This pr adds a json file, similar to versions.tfvars.json, to allow to programmatically change parameters using jq.
We could use versions.tfvars.json for that, but I think it would be better to separate them

### CODE REVIEW

**Comment by lzianekhodja** [comment]
Cloud you show an example how do you expect to use this file (extra.tfvars.json) ?

**Comment by dbrasseur-aneo** [comment]
PR on Action.Deploy soon as an example (and something I need)

File: infrastructure/quick-deploy/aws/all/Makefile [hunk 1]
```diff
@@ -2,6 +2,7 @@ CURRENT_DIR:=$(shell pwd)
 GENERATED_DIR=$(CURRENT_DIR)/generated
 INGRESS_CERTIFICATES_DIR=$(GENERATED_DIR)/certificates/ingress
 PARAMETERS_FILE?=parameters.tfvars
+EXTRA_JSON?=../../../../extra.tfvars.json
```
Comment (lzianekhodja) [inline infrastructure/quick-deploy/aws/all/Makefile:5]:
```suggestion
EXTRA_PARAMETERS_FILE?=../../../../extra.tfvars.json
```

File: infrastructure/quick-deploy/localhost/all/Makefile [hunk 2]
```diff
@@ -2,6 +2,7 @@ CURRENT_DIR:=$(shell pwd)
 GENERATED_DIR=$(CURRENT_DIR)/generated
 INGRESS_CERTIFICATES_DIR=$(GENERATED_DIR)/certificates/ingress
 PARAMETERS_FILE?=parameters.tfvars
+EXTRA_JSON?=../../../../extra.tfvars.json
```
Comment (lzianekhodja) [inline infrastructure/quick-deploy/localhost/all/Makefile:5]:
```suggestion
EXTRA_PARAMETERS_FILE?=../../../../extra.tfvars.json
```

**Review by lzianekhodja — APPROVED** [review]

**Pull request merged**